utils/speed-regression.tcl

```diff
@@ -20,15 +20,15 @@ proc run-tests branches {
         exec -ignorestderr make 2> /dev/null
 
         if {$branch_id == 0} {
-            puts "  copy redis-benchmark from unstable to /tmp..."
-            exec -ignorestderr cp ./redis-benchmark /tmp
```

Comment:
Bold statement by me, maybe we just delete this? Has anybody ever run this? I've never seen this before.

Reply:
Let me check if this file is necessary. I never use it before.

Reply:
I run the daily CI without this file, all test cases pass. I think we could remove it safely.  (Here is the result https://github.com/hwware/redis/actions/runs/8454862841)

Reply:
Sounds good, let's delete it.